go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	github.com/stackrox/external-network-pusher v0.0.0-20210419192707-074af92bbfa7
 	github.com/stackrox/helmtest v0.0.0-20220930104945-c4a3c15e834a
 	github.com/stackrox/k8s-istio-cve-pusher v0.0.0-20210422200002-d89f671ac4f5
-	github.com/stackrox/scanner v0.0.0-20220929215850-334d4658d5e6
+	github.com/stackrox/scanner v0.0.0-20221122151333-4c8cbd61bfa7
```

Comment:
just confirming this is the master branch, right?

Reply:
> just confirming this is the master branch, right?

Yes.